### DISCUSSION TITLE
[Question] Option to filter host's DNA

### DISCUSSION THREAD

**Discussion by matrs** [franciscozorrilla/thesis_pipeline#38] [Q&A] [answered]
Hello, 
I was wondering if  you haven't tried to check for host DNA in the input files. I've seen some reports showing that it can be problematic for downstream analysis. I'm new analyzing metagenomes, so I'm not sure about it and maybe for your particular analysis this isn't a concern.  Searching for software to do this,  I found `kneaddata`, which separates bacterial DNA from host DNA, although  it seems to include trimming too, which maybe isn't the most convenient. 

https://huttenhower.sph.harvard.edu/kneaddata/

Jose Luis

**Comment by franciscozorrilla** [comment, marked as answer]
Hi Jose,

Thank you for your suggestions, `KneadData` or something similar may be a good addition for user cases where removing contaminants is imperative for analysis. Btw, if you want to filter out eukaryotic DNA you can also run `EukRep` on your assebmlies as shown [here](https://github.com/patrickwest/EukRep_Pipeline) or see #14 . Having said that, `KneadData` would still be useful for removing eukaryotic reads from humans or other species.

Francisco

**Comment by matrs** [comment]
I see, that information about `euKRep` it's useful, I'll check it. 

Thanks!
